strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupId/LowercaseValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+import org.carlspring.strongbox.providers.ProviderImplementationException;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.validation.exceptions.LowercaseValidationException;
+
+import java.io.IOException;
+
+/**
+ * Created by dinesh on 12/6/17.
+ */
+public interface LowercaseValidator
+{
+
+    /**
+     * This method checks if the groupId component of the path is in lowercase
```

Comment:
Since you made this class more generic, this line:
```
     * This method checks if the groupId component of the path is in lowercase
```
will have to be changed to:
```
     * This method checks if the respective part of the path is lowercase
```

Reply:
Done.